packages/venia-concept/venia-upward.yml

```diff
@@ -119,6 +122,26 @@ favicon:
         inline: ./media/favicon.ico
       encoding: binary
 
+index:
+  inline:
+    status: 200
+    headers:
+      inline:
+        content-type:
+          inline: text/html
+    body:
+      engine: mustache
+      template: './templates/index.mst'
+      provide:
+        isDevelopment:
```

Comment:
Since this logic is now used in more than one place, you can set `isDevelopment` as a ConditionalResolver in the top level context.